javascript/javascript2/week2/script.js

```diff
@@ -0,0 +1,99 @@
+// WARMUP ARRAY EXERCISES
+// 1.1. Doubling of number
+
+const numbers = [1, 2, 3, 4];
+
+const doubleOddNumbers = function (arrayOfNumbers) {
+  const newArrayOfNumbers = arrayOfNumbers
+    .filter((number) => number % 2 !== 0)
+    .map((number) => number * 2);
+  console.log("The doubled numbers are: ", newArrayOfNumbers);
+};
+doubleOddNumbers(numbers);
+
+// 1.2. Working with movies
+// 1-short title
+const shortTitleMovies = movies.filter((movie) => movie.title.length <= 5);
+console.log(shortTitleMovies);
+
+// 2-long titles
+const longTitleMovies = movies.filter((movie) => movie.title.length > 50);
+console.log(longTitleMovies);
+
+// 3-Count the number of movies made between 1980-1989
+const the80sMovies = movies.filter(
+  (movie) => movie.year >= 1980 && movie.year <= 1989
+).length;
+console.log(the80sMovies); // 638
+
+// 4-Create a new array that has an extra key called tag. The tag is based on the rating: Good (>= 7), Average (>= 4 and < 7), Bad (< 4)
+const moviesWithTag = movies.filter((movie) => {
+  if (movie.rating >= 7) {
+    return (movie.tag = "Good");
+  }
+  if (movie.rating >= 4) {
+    return (movie.tag = "Average");
+  } else {
+    return (movie.tag = "Bad");
+  }
+});
+// Using Ternary Operator
+// let rating =
+//     (movie.rating >= 7) ? "Good" : movie.rating >= 4 && movie.rating < 7 ? "Average" : "Bad";
```

Comment:
movie.rating >= 7 ? "Good" : movie.rating >= 7 ? "Average" : "Bad";

I think this is what you're looking for.
[This article](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator#conditional_chains),  will help you to understand clearly.